add-steps/README.md

```diff
@@ -0,0 +1,76 @@
+# Go faster by adding more steps
+
+This directory contains a docker-compose environment that runs a demo that you
+can use to test linkerd's performance. The results of
+the demo are discussed in much more detail in Buoyant's blog post, [Go faster by adding more steps](
+https://blog.buoyant.io/2017/01/25/go-faster-by-adding-more-steps/).
+
+## Startup
+
+The [`docker-compose.yml`](docker-compose.yml) file that's included in this
+directory is configured to run the demo. Start all of the containers with:
+
+```bash
+$ docker-compose build && docker-compose up -d
+```
+
+That command will build and run all of the following containers:
+
+![add-steps](add-steps.png)
+
+* **2 slow_cookers**: Traffic is generated using [slow\_cooker](
+https://github.com/BuoyantIO/slow_cooker). The slow\_cookers are configured to
+send 300 requests per second to both the linkerd cluster and the baseline
+cluster, both of which are fronted by HAProxy.
+
+* **HAProxy**: HAProxy handles all incoming-traffic. It is configured to listen
```

Comment:
Yeah, went back and forth on this. I'm trying to make this post about linkerd vs. no linkerd, and did not want to call out HAProxy as a distraction.